addOns/pscanrules/src/test/java/org/zaproxy/zap/extension/pscanrules/ContentSecurityPolicyScannerUnitTest.java

```diff
@@ -24,32 +24,54 @@
 
 import org.junit.jupiter.api.Test;
 import org.parosproxy.paros.core.scanner.Alert;
+import org.parosproxy.paros.core.scanner.Plugin.AlertThreshold;
+import org.parosproxy.paros.network.HttpHeader;
 import org.parosproxy.paros.network.HttpMalformedHeaderException;
 import org.parosproxy.paros.network.HttpMessage;
 
 public class ContentSecurityPolicyScannerUnitTest
         extends PassiveScannerTest<ContentSecurityPolicyScanner> {
 
+    private static final String REASONABLE_POLICY =
+            "default-src 'self'; script-src 'self' "
+                    + "storage.googleapis.com cdn.temasys.io cdn.tiny.cloud *.google-analytics.com; "
+                    + "style-src 'self' *.googleapis.com; font-src 'self' data: *.googleapis.com "
+                    + "fonts.gstatic.com; frame-ancestors 'none'; worker-src 'self'; form-action 'none'";
+    private static final String HTTP_HEADER_CSP = "Content-Security-Policy";
+
     @Override
     protected ContentSecurityPolicyScanner createScanner() {
         return new ContentSecurityPolicyScanner();
     }
 
     @Test
-    public void exampleBadCsp() throws HttpMalformedHeaderException {
+    public void shouldNotRaiseAlertOnNonHtmlsAtMediumThreshold() {
         // Given
-        HttpMessage msg = new HttpMessage();
-        msg.setRequestHeader("GET https://www.example.com/test/ HTTP/1.1");
+        HttpMessage msg = createHttpMessage();
```

Comment:
I'd expect this message to have the same CSP has the following test, otherwise we are not checking just the alert threshold.